safeeyes/plugins/donotdisturb/plugin.py

```diff
@@ -63,7 +63,7 @@ def is_active_window_skipped():
                     if process in skip_break_window_classes:
                         return True
                     elif process in take_break_window_classes:
-                        if is_fullscreen and unfullscreen_allowed:
+                        if is_fullscreen and unfullscreen_allowed and not pre_break:
```

Comment:
Hi @di72nn,
I wonder why you are using this `pre_break` check. Can't we simply call this function as we did earlier in `on_start_break`?

Reply:
I think that "unfullscreening" should happen only with the actual break, not 10 seconds before it (or whatever amount of time is configured). However, I have no strong opinion in regard to that.

Reply:
You are right that makes sense :-)